### DISCUSSION TITLE
lua 코딩 테스트

### DISCUSSION THREAD

**Discussion by seong0929** [Tiny-Metro/Poly-Metro#307] [Q&A]
2023.09.20에 머지한 develop 브랜치에서 테스트한 결과
## Event
1. 치트 코드로 Event 발생 시 Event 창에 표시가 되지 않음

## Invenst
1. 투자 미션 성공 시 하루가 지나야 사라짐
2. 투자 미션 사라진 후(2주차)가 되자마자 투자 미션 2개가 생성됨
3. 1010 투자 미션('180명을 수송하세요'): 게임 플레이를 못 하여 성공을 할 수 없는 상황에서 1주일이 지나자 성공이 뜸.

## etc
1. 카메라 시작의 startporint의 position의 z 좌표가 235로 수정하는 거 맞나요? 화면으로 확인하기 힘들어요.
2. 돈이 실시간으로 오르지 않고, 하루마다 오름

[치트 코드 노션](https://www.notion.so/9a5d9014dd094423a8ea80a3723dcb4a)

**Comment by seong0929** [comment]
치트 코드가 작동이 되지 않는 것 같습니다.
1. skip_time과 skip_time 14 이 실행되지 않음(게임 실행 후 "`"클릭 후 함수 입력, 혹은 게임 실행 후 게임 기능의 일시정지 후 함수 입력 두 경우 모두 실행이 되지 않음.)
2. money 1000 실행 해도 소지금 변화가 없음.
3. compain_off는 작동이 되기는 하는 듯 싶었으나, 2주차에서 게임 오버 등장하였습니다.

**Comment by mille-feu** [comment]
이벤트
1. 치트 코드는 다시 살펴보도록 하겠습니다.

투자 미션
1. 성공한 투자 미션이 다음날에 바로 사라지나요?
2. 2주차에는 투자 미션이 2개가 생성되는게 맞지않나요?
3. 조금 자세한 설명이 필요합니다.

**Reply by seong0929** [reply]
투자 미션
1. 네
2. 1주차때 미션 한 개 성공 후 2주차 진입 시 2개가 새로 생긴 상황입니다.

[이 위키](https://github.com/Tiny-Metro/Tiny-Metro/wiki/Mod-scripting-guide-:-Sample-code)를 보니 개선 사항을 찾았습니다. 수정을 해보겠습니다.
